cgi.go

```diff
@@ -195,6 +196,7 @@ func addHeadersToServer(fc *frankenPHPContext, trackVarsArray *C.zval) {
 func addPreparedEnvToServer(fc *frankenPHPContext, trackVarsArray *C.zval) {
 	for k, v := range fc.env {
 		C.frankenphp_register_variable_safe(toUnsafeChar(k), toUnsafeChar(v), C.size_t(len(v)), trackVarsArray)
+		C.frankenphp_register_env_safe(toUnsafeChar(k), toUnsafeChar(v), C.size_t(len(v)))
```

Comment:
Can't we pass an additional flag to `frankenphp_register_variable_safe()` instead of introducing a new function? This would avoid a lot of cgo calls.

By the way, even if unrelated, we could do a single call to `frankenphp_register_variable_safe()` by looping C-side for the same reason (likely a micro-optimization, but still).